### ISSUE TITLE
<div> content is ignored

### ISSUE THREAD

**Issue by HarHarLinks** [vector-im/element-web#17612]
### Description

```
{
  "type": "m.room.message",
  "content": {
    "msgtype": "m.text",
    "format": "org.matrix.custom.html",
    "body": "<div>within div</div> without div",
    "formatted_body": "<div>within div</div> without div"
  }
}
```

![image](https://user-images.githubusercontent.com/2803622/121500586-bb771b80-c9de-11eb-8c5c-0f2d1ed4d815.png)

### Steps to reproduce

- `/html` or any other means of creating above event

Element can ignore the structure given by the tag, and it's other properties, but should at least show the content?
See also parity with android, which does show it as expected.

Possibly Element could warn about tags not being supported when sending `/html`.

Logs being sent: no

### Version information

- **Platform**: desktop

For the desktop app:

- **OS**: Arch
- **Version**: aur/element-desktop-nightly-bin 2021061001-1

**Comment by jaywink** [comment]
Just hit this and but none of my colleagues I asked could see the issue.

I went through my labs flags and turning off LaTeX rendering made the content inside div's visible. @HarHarLinks does this solve your issue too, making the LaTeX rendering the fault here?

My version is Element Nightly version: 2021111801

**Comment by HarHarLinks** [comment]
You are correct, turning off latex in labs reveals the text between the div tags. it produces an unexpected linebreak then with an even source as given above:
![image](https://user-images.githubusercontent.com/2803622/142611171-73d17cb2-bdfd-421d-96a0-21157f1c0db8.png)
